README.md

```diff
@@ -1,24 +1,39 @@
-# Frobnicator (this is a template)
+# na pewno bot for Twitter ([@napewnobot](https://twitter.com/napewnobot))
```

Comment:
niezbyt widać, żeby ten bot rzeczywiście działał...

Reply:
Póki co bot celowo jest wyłączony. W obecnym stanie bardzo szybko otrzymałby shadowbana ze względu na ilość requestów i przede wszystkim takie same, generyczne odpowiedzi. Po shadowbanie użytkownicy np. nie dostawaliby powiadomień o odpowiedziach bota (dostał takiego shadowbana po pierwszym uruchomieniu, shadowban zniknął dopiero niedawno). Uruchomię go dzisiaj, po jego ukończeniu.

Reply:
Bot już został uruchomiony. Żeby nie dostać shadowbana i nie przekroczyć limitu, który tak naprawdę nikomu nie jest znany (wiadomo tylko, że można pisać 2400 Tweetów dziennie i jest to podzielone na mniejsze okresy czasu, ale czy jest to podzielone jednostajnie - tak naprawdę niewiadomo) nie działa on w nieskończonej pętli - jest raczej przeznaczony do "puszczenia" kilka razy dziennie. Tweety pisze co minutę.